uvtools/plot.py

```diff
@@ -723,3 +723,185 @@ def plot_diff_uv(uvd1, uvd2, pol=None, check_metadata=True, bins=50):
         fig.colorbar(cax)
 
     return fig
+
+def plot_diff_1d(uvd1, uvd2, antpairpol, plot_type="both", 
+                 check_metadata=True, dimension=None):
+    # TODO: docstring
```

Comment:
Can be removed.

Reply:
Guess I didn't push the commit that removed this! So it's done.